lib/type.ml

```diff
@@ -223,3 +223,34 @@ let rec of_absform = function
      of_elem (TySingleton (Atom "not_implemented"))
   | F.TyPredef {line; name; args} ->
      failwith (!%"Prease report: line:%d: unexpected predef type: '%s/%d'" line name (List.length args))
+
+let rec of_erl_type = function
+  | Erl_type.Any -> TyAny
+  | None -> TyBottom
+  | Unit -> TyBottom
```

Comment:
The type corresponding to `Unit` (`no_return()` ) is not `TyBottom`. Please do not implement yet for this case.

Reply:
I fixed to treat `Unit` as unimplemented.